### DISCUSSION TITLE
react-aria ie 11 support

### DISCUSSION THREAD

**Discussion by bugzpodder** [adobe/react-spectrum#3338] [Q&A] [answered]
since react-aria are just hooks there should be no issues when using it in ie 11?

**Comment by reidbarber** [comment, marked as answer]
IE11 isn't supported: https://react-spectrum.adobe.com/react-aria/interactions.html#supported-browsers-and-platforms

There are definitely newer ES features used that won't work in IE11, and we don't polyfill to support IE.

**Reply by bugzpodder** [reply]
thanks. would it work if i transpile node_modules using next_transpile_modules

**Reply by devongovett** [reply]
Not likely. We rely on DOM APIs that don't exist in IE 11. You could try to polyfill those two but no guarantees.

I'd just drop IE if you can. Microsoft doesn't even support it anymore.
